pkg/kubelet/dockertools/docker.go

```diff
@@ -534,19 +533,29 @@ var (
 	ErrContainerCannotRun = errors.New("Container cannot run")
 )
 
-func inspectContainer(client DockerInterface, dockerID, containerName, tPath string) (*api.ContainerStatus, error) {
+// Internal information kept for containers from inspection
+type containerStatusResult struct {
+	status api.ContainerStatus
+	podIP  string
```

Comment:
This is indeed PodIP information, and only obtained through network container. We used to expose such information to master through extra container entry called POD in PodStatus. I have cleaned that code by pushing down entire PodStatus to kubelet, and hide POD container completely. This code is goging further to remove PodIP from api.ContainerStatus.